src/airflow/Dockerfile

```diff
@@ -1,29 +1,7 @@
 FROM apache/airflow:2.7.1-python3.8
```

Comment:
Our project is in Python 3.10, is this correct?

Reply:
Yes, that's right! While I don't think it would affect things much (since Airflow only interprets the DAG files, and they are pretty light in terms of syntax), it's good to have things consistent all the way through. I've now changed this